project/MultiNode.scala

```diff
@@ -51,7 +51,7 @@ object MultiNode extends AutoPlugin {
     // -Dmultinode.Djava.net.preferIPv4Stack=true -Dmultinode.Xmx512m -Dmultinode.XX:MaxPermSize=256M
     // -DMultiJvm.akka.cluster.Stress.nrOfNodes=15
     val MultinodeJvmArgs = "multinode\\.(D|X)(.*)".r
-    val knownPrefix = Set("multnode.", "akka.", "MultiJvm.")
+    val knownPrefix = Set("akka.", "MultiJvm.")
```

Comment:
In my quest to find out what was wrong, I found this typo. Note the missing `i`. 
This code is here since 2014 and nobody ever complained about this typo. Removing it. 

btw, I do think the regex `multinode\\.(D|X)(.*)` is also not used. I couldn't find any configuration in our build passing `multinode.D` or `multinode.X`. Probably something we were doing in Jenkins, but now obsolete. 

I can remove it on another PR. I don't want to get this now failing for unknown reasons.